src/Compiler/Driver/CompilerDiagnostics.fs

```diff
@@ -566,7 +565,9 @@ module OldStyleMessages =
     let ValNotLocalE () = Message("ValNotLocal", "")
     let Obsolete1E () = Message("Obsolete1", "")
     let Obsolete2E () = Message("Obsolete2", "%s")
-    let ExperimentalE () = Message("Experimental", "%s")
+    let Experimental1E () = Message("Experimental1", "")
```

Comment:
For my general understanding, what are the three shapes of this thing meant to represent? Also, Experimental1E and Experimental3E are the same...

Reply:
@psfinaki Experimental1E and Experimental3E are not same.

```xml 
<data name="Experimental1" xml:space="preserve">
    <value>This construct is experimental</value>
</data>
<data name="Experimental2" xml:space="preserve">
    <value>. {0}</value>
</data>
  <data name="Experimental3" xml:space="preserve">
    <value>. This warning can be disabled using '--nowarn:57' or '#nowarn "57"'.</value>
</data>
```

Reply:
Oh I see, thanks!